tox.ini

```diff
@@ -1,11 +1,17 @@
 [tox]
 envlist =
     lint
-    py{35,36,37}-django111
-    py{35,36,37,38}-django22
```

Comment:
Referring to [this comment in the pystac PR](https://github.com/azavea/pystac/pull/108/files#r451660113), I've preemptively dropped support for Python 3.5 before it hits EOL. I recognize that this package might have a slightly wider audience and this decision could have more impact. I can easily add 3.5 back in if dropping it is premature.

Reply:
We can, but I think doing so bundles in making sure anything related to `future-fstrings` is accounted for. It might make sense to defer that to its own issue.

Reply:
Good call. I'll revert this and we can drop support in #18.